newsfragments/198.feature

```diff
@@ -0,0 +1 @@
+New tool dev.dxtbx.show_mask_info added: can pass in data and it will show the number of masked pixels for each module 
```

Comment:
I would suggest either removing this newsfragment or moving the command out of the `dev.dxtbx` namespace. Perhaps the latter, since you have added a test for it?

Reply:
👍

Reply:
Done in dcc47ecf235b32d2566b194de5a188f3a8867a9a